gazelle/pythonconfig/pythonconfig.go

```diff
@@ -136,6 +140,7 @@ type Config struct {
 	libraryNamingConvention      string
 	binaryNamingConvention       string
 	testNamingConvention         string
+	extraVisibility              []string
```

Comment:
I could have sworn I did... Sorry about that, I definitely meant to.

...
...

Ah, looks like I stashed the changes instead of committing them. /facepalm. Anyway, updated 👍.